src/connector/src/sink/iceberg/mod.rs

```diff
@@ -1366,8 +1559,189 @@ impl SinkCommitCoordinator for IcebergSinkCommitter {
         }
 
         tracing::info!("Succeeded to commit to iceberg table in epoch {epoch}.");
+
+        if self.is_exactly_once {
+            self.delete_row_by_sink_id_and_end_epoch(&self.db, self.sink_id, epoch)
+                .await?;
+            tracing::info!("Succeeded delete pre commit metadata in epoch {epoch}.");
+        }
```

Comment:
For better observability, I suggest using the additional field `status` and mark it as `committed` instead of deleting it immediately. We can delete the log asynchronously.

Reply:
BTW, for dropping sink, we need to delete the commit log as well.